docs/UserGuide.md

```diff
@@ -28,10 +28,12 @@ guest management tasks done faster than traditional GUI apps.
 1. Type the command in the command box and press Enter to execute it. e.g. typing **`help`** and pressing Enter will open the help window.<br>
    Some example commands you can try:
 
-   * **`list`** : Lists all guests.
-
    * **`add`**`n/John Doe p/98765432 e/johnd@example.com rm/05-73 dr/13/09/22 - 15/09/22 ng/3`: Adds a guest named `John Doe` to the guest book.
 
+   * **`find`**`John` : Finds any guest that has `John` (Case insensitive) in any of their details.
```

Comment:
Should we user lowercase `case` instead of `Case`?

Reply:
Agreed and changed.